tango/step_info.py

```diff
@@ -157,7 +157,7 @@ def to_json_dict(self) -> Dict[str, Any]:
                 if isinstance(v, set)
                 else v
             )
-            for k, v in asdict(self).items()
+            for k, v in sorted(asdict(self).items(), key=lambda x: x[0])
```

Comment:
Why sorted?

Reply:
I was trying to make it deterministic so we get the same digest if it hasn't changed, but I just realized we'll need to return `OrderedDict` for that to work 🤦‍♂️ 0eba6ab